src/forms/services/form-submission.service.ts

```
@@ -166,6 +180,141 @@ export class FormSubmissionService {
+
+          // Find matching submitted field value
+          const matchingField = createFormSubmissionDto.customFields.find(
+            (f) => f.fieldId === fieldId
+          );
+
+          if (matchingField) {
+            // Fetch the field name using fieldsService
+            let fieldName = fieldId;
+            try {
+              const fieldDef = await this.fieldsService.getFieldById(fieldId);
+              if (fieldDef && fieldDef.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)